src/components/LocationBias.tsx

```diff
@@ -79,26 +79,31 @@ export function LocationBias({
     executeSearch(answersActions);
   }
 
+  const screenIsLarge = window.innerWidth >= 1024;
+
   return (
-    <div className={cssClasses.container}>
-      {isFetchingLocation && <div className={cssClasses.loadingIndicatorContainer}/>}
+    <div className={cssClasses.locationBiasContainer}>
+      {screenIsLarge && isFetchingLocation && <div className={cssClasses.loadingIndicatorContainer}/>}
       <span className={cssClasses.location}>
         {locationBias.displayName}
       </span>
       <span className={cssClasses.source}>
         {attributionMessage}
       </span>
```

Comment:
in percy, I see that there's a new space gap between `(base on your device) -` and `update your location`, can we remove that?
![Screen Shot 2022-07-07 at 10 07 06 AM](https://user-images.githubusercontent.com/36055303/177793739-fa569b71-75ba-4455-928b-ee62bfedb21c.png)
![Screen Shot 2022-07-07 at 10 07 08 AM](https://user-images.githubusercontent.com/36055303/177793747-eccc9d5b-4fda-47af-a9e3-41855db0a947.png)